.travis.yml

```diff
@@ -5,6 +5,7 @@ python:
     - "3.4"
 
 env:
+    - SECRET_KEY=djkk\&8izc3tmgva\(m\$%u_zvjk8%f@-6w\(d-z68@m-@sh3nz\#pi
     - DJANGO_VERSION=1.8
 
 install:
```

Comment:
probably under install you have to add `gem install travis`. the applicatin travis seems to be a ruby program (and oddly enough not included on the travis build machines)